tfdiags/contextual_test.go

```diff
@@ -565,3 +569,7 @@ func TestGetAttribute(t *testing.T) {
 		t.Fatalf("paths don't match:\nexpected: %#v\ngot: %#v", path, p)
 	}
 }
+
+func TestDiagnosticAddress(t *testing.T) {
+
```

Comment:
I guess this might be intended to test that we don't overwrite an address which has been set?

Reply:
yes it was! I meant to get that covered in the main test above, and delete this ;)